addons/Shooting_Range/src/presenter.js

```diff
@@ -616,7 +620,11 @@ function AddonShooting_Range_create() {
 
         if (state.isStarted) {
             presenter.changeLevel(state['actualLevel']);
-            presenter.getActualLevel().start(state['actualLevelTimeElapsed'], state['clickedElements']);
+            presenter.getActualLevel().start(
+                state['actualLevelTimeElapsed'],
+                state['clickedElements'],
+                state['wrongedElements'],
```

Comment:
Dla starych lekcji w mCourserze tych elelemntów nie ma w stanie, więc dostaniesz tutaj undefined. Spowoduje to błąd przy otwieraniu starych lekcji.